charts/sonarr/Chart.yaml

```diff
@@ -1,17 +1,20 @@
 apiVersion: v2
-appVersion: 3.0.3.911
-description: Sonarr is a television show downloading client
 name: sonarr
-version: 5.0.1
+description: Sonarr Chart
```

Comment:
Suggest we have a more descriptive description other than "'chart name' chart" - was there a problem with the original description?